tests/bcl-test/BCLTests/templates/tvOSApp/ViewController.cs

```diff
@@ -54,7 +56,26 @@ protected virtual void TerminateWithSuccess ()
 		}
 #endif
 
-		public override void ViewDidLoad ()
+		public async Task<string[]> GetSkippedTests ()
```

Comment:
It looks like the code in this method is repeated several times, maybe add an `IgnoreFileParser.ParseAssemblyResource (Assembly)` method:

```csharp
class IgnoreFileParser {
    public async Task<string[]> ParseAssemblyResourcesAsync (Assembly asm)
    {
        var ignoredFiles = new List<string> ();
        // the project generator added the required resources,
        // we extract them, parse them and add the result
        foreach (var resourceName in asm.GetManifestResourceNames ()) {
            if (resourceName.EndsWith (".ignore", StringComparison.Ordinal)) {
                using (var stream = executingAssembly.GetManifestResourceStream(resourceName))
                    using (var reader = new StreamReader (stream)) {
                        var ignored = await ParseStream (reader);
                        // we could have more than one file, lets add them
                        ignoredFiles.AddRange (ignored);
                    }
                }
            }
        return ignoredFiles.ToArray ();
    }
}
```